### PR TITLE
add permission on operation

### PR DESCRIPTION
**Pull request by Sispheor**
- Fix UI
- add perm on operation
- bulk perm update from service page and portfolio page
- drop compose v2 support

#724

### CODE REVIEW

**Review by EliasBoulharts — APPROVED** [review]

File: dev.docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 # add this file to the docker compose execution when developing Squest
-version: '3.7'
-
 services:
```
Comment (a-belhadj) [inline dev.docker-compose.yml:2]: CHANGELOG Breaking changes does not support docker compose v1 anymore

File: service_catalog/forms/form_utils.py [hunk 2]
```diff
@@ -1,12 +1,10 @@
+import inspect
```
Comment (a-belhadj) [inline service_catalog/forms/form_utils.py:1]: ?

File: service_catalog/models/instance.py [hunk 3]
```diff
@@ -116,6 +115,8 @@ def __str__(self):
     @property
     def docs(self):
         filtered_doc = list()
+        if self.service is None:
```
Comment (a-belhadj) [inline service_catalog/models/instance.py:118]: service shouldn't be None, it should be required in Instance

File: service_catalog/views/instance.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import logging
+from gc import enable
```
Comment (a-belhadj) [inline service_catalog/views/instance.py:2]: ?

File: tests/test_service_catalog/test_forms/test_admin_service_forms.py [hunk 5]
```diff
@@ -11,22 +12,27 @@ def setUp(self):
...
             {
                 "name": "new_service_2",
                 "description": "a new service 2",
+                "permission": FormUtils.get_default_permission_for_operation(),
             },
             {
                 "name": "new_service_3",
-                "description": "a new service 3"
+                "description": "a new service 3",
+                "permission": FormUtils.get_default_permission_for_operation(),
```
Comment (a-belhadj) [inline tests/test_service_catalog/test_forms/test_admin_service_forms.py:25]: Three times the same

**Review by a-belhadj — CHANGES_REQUESTED** [review]
small changes to do before merge

**Review by a-belhadj — APPROVED** [review]

**Pull request merged**